client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/index.js

```diff
@@ -56,7 +56,7 @@ const renderSummary = (
 	}
 	str += 'US' === country ? postcode.split( '-' )[ 0 ] : postcode;
 	if ( showCountry ) {
-		str += ', ' + countriesData[ country ].name;
+		str += ', ' + ( countriesData[ country ] ? countriesData[ country ].name : country );
```

Comment:
The `renderSummary` block overall is hard to read. Would be cool to split at least this line into a variable and the concatenation for easier readability, something like this:

```
const countrySummary = ( countriesData[ country ] ? countriesData[ country ].name : country );
str = [str, countrySummary].join(', ');
```